crypter.go

```
@@ -445,6 +445,9 @@ func (obj JSONWebEncryption) Decrypt(decryptionKey interface{}) ([]byte, error)
 	if err == nil {
 		// Found a valid CEK -- let's try to decrypt.
 		plaintext, err = cipher.decrypt(cek, authData, parts)
+		if err != nil {
+			println("square/go-jose [WARN]: error decrypting cipher:" + err.Error())
```

@Baha-sk you should return this error

---

I'm not going to change the library's behaviour as they did not return it originally.. I'm simply printing out to help support it.

---

oh wow... gotcha.